src/reducers/filter.js

```diff
@@ -0,0 +1,19 @@
+import actions from '../actions/index';
+
+const { action } = actions;
+const { CHANGE_FILTER } = action;
+
+const filterReducer = (state = '', action) => {
```

Comment:
Please make sure the default state for the filter reduce is 'All' [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png)